.github/workflows/test_suite.yml

```diff
@@ -39,6 +47,7 @@ jobs:
           python -m pip uninstall -y goalie
           python -m pip install -e .
       - name: Test Goalie
+        if: ${{ steps.changed-files.outputs.any_changed == 'true' }}
```

Comment:
Do you not also need to check if this is a cron job? i.e., the regular test suite runs on Sunday early mornings?

Reply:
I'm pretty sure what I did in https://github.com/mesh-adaptation/mesh-adaptation-docs/pull/33 is what you need.

Reply:
Thanks! Done in 5d2d570cbfb6c89d2fd00f90f199787ca869f3c6